staff/joel-suarez/git/README.md

```diff
@@ -0,0 +1,66 @@
+# GIT Coomands
+
+These are some of my git commands
+
+### How to init a git repository
+```sh
+$ git init
+$ git remote add origin 'github url'>
+$ git add 'folder to add'>
+$ git commit -m "message"
```

Comment:
Add specific sections to define and explain the "add" & "commit" commands 👻

Reply:
Done I added comments to the add & commit commands.

Reply:
🚀🚀😁